vtex-apps/store-components
# Add option to zoom on mouseover on desktop

#### What is the purpose of this pull request?

Allow the user to choose if the zoom will be enabled in on hover or on  click
#### What problem is this solving?

The need for a more dynamic behavior for zoom on desktop.

#### How should this be manually tested?
[workspace on hover](https://zoomonhover--storecomponents.myvtex.com/classic-shoes/p)
[workspace on click](https://zoomonclick--storecomponents.myvtex.com/classic-shoes/p)
[workspace gallery](https://zoomongallery--storecomponents.myvtex.com/classic-shoes/p)
(forced prop value throught code since cms is not persisting changes, but it should work using blocks.json)
#### Screenshots or example usage

#### Types of changes
